src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -116,6 +116,39 @@ def test_ViSession_attribute(device_info):
             print(e)
             pass
 
+
+def test_parameter_count0(device_info):
+    with nidmm.Session(device_info['name']) as session:
+        assert session.get_aperture_time_info()[1] == 0 # Assuming default aperture time unit will be seconds
+
+                
+def test_parameter_count1(device_info):
+    with nidmm.Session(device_info['name']) as session:
+        session.configure_measurement_digits(nidmm.Function.DC_VOLTS, 1, 5.5)
+        session._initiate()
```

Comment:
I don't think there's any good reason to use any private methods ever in system tests.